AdminDashboard/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ConnectionStrings": {
```

Comment:
Let me know if we want to gitignore appsettings.json for more security - then any developer can just copy the configs locally!

Reply:
is there an "easy" / built-in way to read this from an environment variable instead?